### PR TITLE
Issue #775 Drop Frameback

### PR DESCRIPTION
**Pull request by szhou8813**
No one uses frameback, so we're dropping it.

### CODE REVIEW

File: packages/react-server-test-pages/pages/navigation/playground.js [hunk 1]
```diff
@@ -131,8 +105,6 @@ export default class NavigationPlaygroundPage {
 					<li>CT: Client Transition</li>
 					<li>RD: Reuse DOM</li>
 					<li>BD: Bundle Data</li>
-					<li>FB: Frameback</li>
-					<li><span className='not-available'>FB</span>: Frameback disabled (already in a frame)</li>
```
Comment (gigabo) [inline packages/react-server-test-pages/pages/navigation/playground.js]: There should be some styling (strikethrough, I think) associated with the `not-available` class that can be removed from the CSS, too.

**Comment by gigabo** [comment]
This is awesome @szhou8813!  It makes the history navigation stack management _so much simpler!_

I think there's probably some documentation that will need to be tweaked, too.

File: packages/react-server-test-pages/pages/navigation/playground.js (hunk 1 above)
Comment (szhou8813) [inline packages/react-server-test-pages/pages/navigation/playground.js]: Sure, I ll check that out!

**Comment by szhou8813** [comment]
Oh I guess we ll need to update tutorials/docs on react-server.io too. How do I do that?

**Comment by davidalber** [comment]
The content on react-server.io is queried from GitHub. If you update the appropriate docs in the repository it will be reflected on the site within one minute of merging to master.

**Comment by gigabo** [comment]
Awesome!  Thanks for dealing with this @szhou8813!

**Comment by roblg** [comment]
Added `breaking-change`. I believe it's probably unused, but you never know ;)

**Pull request merged**